Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GripperHoverWrapper.cs

```diff
@@ -52,8 +53,18 @@ internal GripperHoverWrapper(UIElement element, GridSplitter.GridResizeDirection
             _gridSplitterDirection = gridSplitterDirection;
             _gripperCursor = gripperCursor;
             _gripperCustomCursorResource = gripperCustomCursorResource;
-            element.PointerEntered += Element_PointerEntered;
-            element.PointerExited += Element_PointerExited;
+            _element = element;
+            _element.PointerEntered += Element_PointerEntered;
+            _element.PointerExited += Element_PointerExited;
+        }
+
+        internal void UpdateHoverElement(UIElement element)
```

Comment:
We should at least unhook it in the applytemplate (if for instance a completely different template is set)